templates/docs/examples/brochure/_50-50.html

```diff
@@ -1,7 +1,7 @@
 <section class="p-section">
   <div class="p-section--shallow">
     <div class="row">
-      <hr class="p-rule">
+      <hr>
```

Comment:
As this is showcased as the rule component example, please bring `p-rule` class names back in this file.

Reply:
I think the `p-rule` class needs to be re-added here